Dockerfile.windows

```diff
@@ -249,19 +255,87 @@ RUN `
   Remove-Item C:\binutils.zip; `
   Remove-Item C:\gitsetup.zip; `
   `
-  Write-Host INFO: Creating source directory...; `
-  New-Item -ItemType Directory -Path ${GOPATH}\src\github.com\docker\docker | Out-Null; `
+  # Ensure all directories exist that we will require below....
+  $srcDir = """$Env:GOPATH`\$Env:DOCKER_SRC_REL_LOCATION`\$Env:BUNDLES_REL_LOCATION"""; `
+  Write-Host INFO: Ensuring existence of directory $srcDir...; `
+  New-Item -ItemType Directory -Path $srcDir | Out-Null; `
   `
   Write-Host INFO: Configuring git core.autocrlf...; `
-  C:\git\cmd\git config --global core.autocrlf true; `
+  C:\git\cmd\git config --global core.autocrlf true;
+
+WORKDIR ${GOPATH}
```

Comment:
Was there a reason to change to this location here? Looks like the script takes care to going to the correct directory, so changing the `WORKDIR` here, only to change it back to `$GOPATH\src\github.com\docker\docker` after, may not be needed.

Effectively, now we do;

```Dockerfile
WORKDIR ${GOPATH}
RUN Build-GoTestSum
WORKDIR ${GOPATH}\src\docker\docker
```

Reply:
Moved the change of directory to `Env:GOPATH` explicitly in the script and removed the first `WORKDIR` above.